Source/loadsave.cpp

```diff
@@ -7,13 +7,41 @@
 
 DEVILUTION_BEGIN_NAMESPACE
 
-BYTE *tbuff;
-
 bool gbIsHellfireSaveGame;
 int giNumberOfLevels;
 int giNumberQuests;
 int giNumberOfSmithPremiumItems;
 
+inline Uint16 SwapLE(Uint16 in)

```

Comment:
Anonymous `namespace` or `static` for these functions and load/save helper.

`inline` does nothing in a `cpp` file: it's not a hint to inline a function in modern C++ compilers -- it simply tells the compiler that all duplicate function definitions are the same and it's OK to deduplicate.